src/site/content/en/blog/bbc-hsts/index.md

```diff
@@ -0,0 +1,78 @@
+3. The server via HTTPS must negotiate a secure connection with the browser.
+4. The content loads as usual.
+
+While redirects work fine, they can be misconfigured in ways that still allow access to the insecure version of a site. Even if everything _is_ configured properly, there is still a security issue in that the user will still connect over insecure HTTP during the redirect phase, which exposes users to the possibility of dangerous [man-in-the-middle attacks](https://en.wikipedia.org/wiki/Man-in-the-middle_attack).
+
+## Enter HSTS
+
+{% BrowserCompat 'http.headers.Strict-Transport-Security' %}
+
+[HSTS](https://developer.mozilla.org/docs/Web/HTTP/Headers/Strict-Transport-Security) is dictated by the `Strict-Transport-Security` HTTP response header for HTTPS requests. When set, return visits to a website will trigger a special redirect known as a "307 Internal Redirect", which is when the browser handles the redirect logic, rather than the server. This prevents the request being intercepted, since it never leaves the browser, so is more secure. As and added bonus, as there is no network request needed to get the redirect, these types of redirect are extremely fast, and so prevent noticeable latency during an HTTP-to-HTTPS hop.
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/J9vQLJCLmnHQPb6spsRe.png", alt="A 307 internal redirect from HTTP to HTTPS, triggered by an HSTS header. The 307 redirect only takes 2 milliseconds.", width="800", height="97" %}
+</figure>
+
+Similar in syntax to `Cache-Control`'s `max-age` directive, an HSTS header specifies a `max-age` directive. This directive takes a value in seconds that specifies how long the policy is effective for:
+
+```http
+Strict-Transport-Security: max-age=3600
+```
+
+In the above example, the policy should only take effect for an hour.
+
+## Deploying HSTS
+
+{% Aside 'caution' %}
+Take care to read the information in this section **very carefully** before you decide to deploy HSTS! Botching an HSTS deployment can result in serious problems for users!
+{% endAside %}
+
+The main drawback of deploying HSTS is if you're not ready to treat your origin as strictly secure. Let's say you have a number of subdomains you're serving resources from, but maybe not all of them are secure. In this scenario, **an HSTS header could break your website.**
+
+The BBC took the right approach to deploying HSTS. As [Neil Craig](https://twitter.com/tdp_org) mentioned in his tweet, the initial value that was set for bbc.com was `max-age=10`.
+
+This approach means that the policy was only initially effective for ten seconds. This doesn't provide much of a benefit, but the idea is to feel out whether there may be issues with applying HSTS at all. As time goes on, you can increase the policy incrementally and see if issues occur. At the time of this writing, bbc.com is specifying an HSTS policy of `max-age=3600`, and that will almost certainly increase over time.
```

Comment:
They've upped it: https://twitter.com/tdp_org/status/1551513913082216448?s=20&t=Z04sbUm1193SCTvau7jRsw

```suggestion
This approach means that the policy was only initially effective for ten seconds. This doesn't provide much of a benefit, but the idea is to feel out whether there may be issues with applying HSTS at all. As time goes on, you can increase the policy incrementally and see if issues occur. At the time of this writing, bbc.com is specifying an HSTS policy of `max-age=86400`, and that will almost certainly increase over time.
```